query_execution/QueryManagerDistributed.cpp

```diff
@@ -119,6 +119,27 @@ bool QueryManagerDistributed::fetchNormalWorkOrders(const dag_node_index index)
   return generated_new_workorder_protos;
 }
 
+void QueryManagerDistributed::processInitiateRebuildResponseMessage(const dag_node_index op_index,
+                                                                    const std::size_t num_rebuild_work_orders) {
+  // TODO(zuyu): Multiple workers support.
```

Comment:
Have we finalized this terminology somewhere? Using the same term ``worker`` in two contexts is confusing. I would recommend using an unambiguous term, like ``node``.